src/store/home/home.thunks.ts

```diff
@@ -4,8 +4,15 @@ import { TxType } from "@hermeznetwork/hermezjs/src/enums";
 import { getPoolTransactions } from "@hermeznetwork/hermezjs/src/tx-pool";
 
 import * as homeActions from "./home.actions";
```

Comment:
This import should also be absolute now right?

Reply:
Correct. Done.